src/App.js

```diff
@@ -13,6 +13,7 @@ import History from "./pages/WatchHistory/History";
 import Liked from "./pages/LikedVedio/Liked";
 import PlayList from "./pages/playList/PlayList";
 import SinglePlayList from "./pages/playList/SinglePlayList";
```

Comment:
You can make a single index.js file and export these from there. The code will look cleaner